GCMessengerSDKSample/app/src/main/java/com/genesys/gcmessengersdksample/data/accounts.kt

```diff
@@ -0,0 +1,59 @@
+package com.genesys.gcmessengersdksample.data
+
+import com.genesys.cloud.integration.async.AsyncAccount
+import com.genesys.cloud.integration.async.core.UserInfo
+import com.genesys.cloud.integration.async.userInfo
+import com.genesys.cloud.integration.bold.BoldAccount
+import com.genesys.cloud.integration.bot.BotAccount
+import com.genesys.cloud.integration.messenger.MessengerAccount
+import java.util.*
+
+object Accounts {
```

Comment:
since the sample is "messenger chat" only sample, we can discard the use of this file.
BoldAccount and BotAccount are not relevant to this sample. The Async account is of a chat that is not supported even on the Bold legacy. (long story)